code/game/objects/items/food/meat.dm

```diff
@@ -181,6 +181,22 @@
 
 ////////////////////////////////////////////MEATS AND ALIKE////////////////////////////////////////////
 
+
+/obj/item/food/meat/slab/pig
+	name = "raw pork"
+	icon_state = "pig_meat"
+	tastes = list("pig" = 1)
+	foodtypes = MEAT | RAW | GROSS
```

Comment:
should these be GORE rather than GROSS? (it seems like a lot of meat products intended for consumption are not even GORE, so perhaps you could just remove it altogether)

Reply:
I'm not a maintainer so there's no requirement you do anything I suggest, just means that almost nobody will like eating this until it is cooked (which isn't weird, just inconsistent).

Reply:
> Can I just keep it for now because I am already inept as it is at coding and this will take me way too much effort.

its just changing this one line to remove the GROSS flag